lstchain/irf/hdu_table.py

```diff
@@ -0,0 +1,293 @@
+    """
+    Create the event_list BinTableHDUs from the given data
+
+    Parameters
+    ----------
+        Data: DL2 data file
+                'astropy.table.QTable'
+        Run: Run number
+                Int
+        Source_name: Name of the source
+                Str
+    Returns
+    -------
+        Events HDU:  `astropy.io.fits.BinTableHDU`
+        GTI HDU:  `astropy.io.fits.BinTableHDU`
+        Pointing HDU:  `astropy.io.fits.BinTableHDU`
+    """
+    name=Source_name
+
+    lam = 1000 #Average rate of triggered events, taken by hand for now
+    # Timing parameters
+    t_start = data['dragon_time'][0]
+    t_stop = data['dragon_time'][-1]
+    time = Time(data['dragon_time'], format='unix', scale="utc")
+
+    obs_time = t_stop-t_start
+
+    #Position parameters
+    focal = 28 * u.m
+    pos_x = data['reco_src_x'] * u .m
+    pos_y = data['reco_src_y'] * u .m
+    pointing_alt = data['alt_tel'] * u.rad
+    pointing_az = data['az_tel'] *u.rad
+
+    coord = camera_to_altaz(pos_x = pos_x, pos_y=pos_y, focal = focal,
+                    pointing_alt = pointing_alt, pointing_az = pointing_az,
+                    obstime = time)
+    coord_icrs = coord.icrs
+    coord_pointing = camera_to_altaz(pos_x = 0 * u.m, pos_y=0 * u.m, focal = focal,
+                pointing_alt = pointing_alt[0], pointing_az = pointing_az[0],
+                obstime = time[0])
+
+    ##########################################################################
+    ### Event table
+    event_table = QTable(
+            {
+                "EVENT_ID" : u.Quantity(data['event_id']),
+                "TIME" : u.Quantity(data['dragon_time'] * u.s),
+                "RA" : u.Quantity(coord_icrs.ra.deg * u.deg),
+                "DEC" : u.Quantity(coord_icrs.dec.deg * u.deg),
+                "ENERGY" : u.Quantity(data['reco_energy'] * u.TeV)
+            }
+        )
+    ##########################################################################
+    ### GTI table
+    gti_table = QTable(
+        {
+            "START" : u.Quantity(t_start * u.s, ndmin=2),
+            "STOP" : u.Quantity(t_stop * u.s, ndmin=2)
+        }
+    )
+    ##########################################################################
+    ### Adding the meta data
+    ### Event table metadata
+    ev_header = DEFAULT_HEADER.copy()
+    ev_header["HDUCLAS1"] = "EVENTS"
+
+    ev_header["OBS_ID"] = run_number
+    ev_header["TSTART"] = t_start
+    ev_header["TSTOP"] = t_stop
+    ev_header["MJDREFI"] = '40587' #Time('', format='mjd', scale="utc") # Unix 01/01/1970 0h0m0
+    ev_header["MJDREFF"] = '0' #Time('0',format='mjd',scale='utc')
+    ev_header["TIMEUNIT"] = 's'
+    ev_header["TIMESYS"] = "UTC"
+    ev_header["OBJECT"] = name
+    ev_header["TELLIST"] = 'LST-1'
+    ev_header["N_TELS"] = 1
+
+    ev_header["RA_PNT"]=float(coord_pointing.icrs.ra.deg)
+    ev_header["DEC_PNT"]=float(coord_pointing.icrs.dec.deg)
+    ev_header["ALT_PNT"]=round(np.rad2deg(data['alt_tel'][0]),6)
```

Comment:
I would not give `ALT_PNT` for a tracking observation. At the very least, it should be the `mean` alt pointing, not the first entry.